js/activity-works.js

```diff
@@ -41,231 +54,247 @@ define(function (require) {
         var stroke;
         var colors;
         var index;
-        var imagepos = new Array();
-        var myimages = new Array();
-        var bitmaps = new Array();
+        var imagepos = [];
+        var myimages = [];
+        var bitmaps = [];
+        var bitmapLabels = [];
         var pen_bitmap;
+        var Star = "images/star.svg";
+        var Dot = "images/dot.svg";
+        var Pen = "images/pen.svg";
+        var shape = 0;
 
-        var star = [[400, 100], [200, 500], [650, 300], [200, 100], [400, 550],
-                    [600, 100], [150, 300], [600, 500], [400, 50]];
-        var hand = [[175, 150], [200, 125], [280, 220], [330, 200], [425, 25],
-      [440, 175], [540, 50], [480, 210], [620, 100], [520, 280],
-                    [680, 270], [520, 330], [420, 420], [400, 450], [280, 450],
-                    [260, 340], [200, 200]];
-        var shapes = [star, hand];
-        var shape = 0
-
-            function init() {
-                if (window.top != window) {
-                    document.getElementById("header").style.display = "none";
-                }
-                document.getElementById("loader").className = "loader";
-                // create stage and point it to the canvas:
-                canvas = document.getElementById("testCanvas");
-
-                index = 0;
-                colors = ["#828b20", "#b0ac31", "#cbc53d", "#fad779", "#f9e4ad",
-                  "#faf2db", "#563512", "#9b4a0b", "#d36600", "#fe8a00",
-                  "#f9a71f"];
-                oldPt = new createjs.Point(400, 300);
-                midPt = oldPt;
-                oldMidPt = oldPt;
-
-                //check to see if we are running in a browser with touch support
-                stage = new createjs.Stage(canvas);
-
-                // enable touch interactions if supported on the current device:
-                createjs.Touch.enable(stage);
-
-                // keep tracking the mouse even when it leaves the canvas
-                stage.mouseMoveOutside = true;
-
-                // enabled mouse over / out events
-                stage.enableMouseOver(10);
+        // Get things started
+        init();
```

Comment:
Please remove redundant comments like this, they're not useful.

Reply:
done..